tutorials/sns-instances-notification-system/index.mdx

```diff
@@ -0,0 +1,292 @@
+---
+meta:
+  title: Creating a notification system with Scaleway SNS and Instances.
+  description: This tutorial introduces you to using SNS for notifications via a simulated CPU monitoring example with terraform.
+content:
+  h1: Setting a notification system with terraform across instances using Scaleway's Instances and Messaging and Queuing SNS, through a simulated CPU monitoring example.
+  paragraph: This tutorial walks you through creating a notification system between instances, from a simulated CPU monitoring webserver to a notification-receiving webserver.
+categories:
+  - messaging
+  - instances
+tags: messaging instances sns notifications monitoring terraform tutorial subscriber publisher receiver
+dates:
+  validation: 2024-03-08
+  posted: 2024-03-08
+---
+
+## Introduction
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [set up authentication for the Terraform provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs#authentication)
+    You have generated a public [SSH key](https://www.scaleway.com/en/docs/identity-and-access-management/organizations-and-projects/how-to/create-ssh-key/#how-to-upload-the-public-ssh-key-to-the-scaleway-interface) (steps 1 to 6)
```

Comment:
Does the user also need to have downloaded/installed Docker on their machine?

Reply:
No need, it is installed on the instance with the user-data on terraform